example/example.json

```diff
@@ -34,6 +34,13 @@
         "published": {
             "type": "boolean",
             "title": "Published?",
+            "items": {
+                "type": "string",
+                "enum": [
+                    "yes",
+                    "no"
+                ]
+            },
```

Comment:
there's no need to use 'items' here. The filed type is `boolean`, so it has only 2 options by definition.